lib/throttle.js

```diff
@@ -25,6 +25,21 @@ module.exports = (api) => {
     };
   };
 
+  api.metasync.debounce = (
+    timeout,
+    fn,
+    args
+  ) => {
+    let timer = null;
+    return () => {
+      clearTimeout(timer);
+      timer = setTimeout(
+        () => (args !== undefined ? fn(...args) : fn()),
+        timeout
```

Comment:
I'm not sure this behavior is correct at all. The `timeout` should be decreased every invocation until it vanishes to be restored with initial value anew, shouldn't it? Your implementation will effectively result in the debounced function never being called if there's a dense stream of events (which is, I believe, what debounced functions are for, fwiw).